### DISCUSSION TITLE
Has anyone else gotten this error: pathspec error with git commit

### DISCUSSION THREAD

**Discussion by gunnardesantisUCONN** [cooperrc/computational-mechanics#188] [Q&A]
Hey yall

When trying to commit the changes from JupyterNotebook work folder to GitHub repo, I get this error every time.  This has happened in both instances that the tutorial references 1)README.md and 2) M01-01 Notes.  Have any of you had this issue? I am worried I will lose work that I do for this course.

(base) jovyan@d4c0cc08fa92:~/work/computational-mechanics$ git add .
(base) jovyan@d4c0cc08fa92:~/work/computational-mechanics$ git commit - m "notes added to M01-01"
error: pathspec '-' did not match any file(s) known to git
error: pathspec 'm' did not match any file(s) known to git
error: pathspec 'notes added to M01-01' did not match any file(s) known to git

Gunnar

**Comment by tad12002** [comment]
Is there a space between - and m? I made that mistake and it gave me that error.

**Comment by gunnardesantisUCONN** [comment]
Ah mate, thanks for getting me out of the pinch! That was the issue.
